taipy/gui/_renderers/factory.py

```diff
@@ -215,6 +215,8 @@ class _Factory:
                 ("width", PropertyType.string_or_number),
                 ("height", PropertyType.string_or_number),
                 ("hover_text", PropertyType.dynamic_string),
+                ("ref_id", PropertyType.dynamic_string),
```

Comment:
I suspect we will want an anchor_type (??) as well: center/below/...

Reply:
not sure